### ISSUE TITLE
dApp is not showing on iOS metamask browsing

### ISSUE THREAD

**Issue by drselump14** [raiden-network/light-client#2710]
## Steps to Reproduce
1.  Open lightclient.raiden.network on iOS metamask browser and safari browser

## Expected Result
- It should show the dApp

## Actual Result
- The page shows an error message "We're sorry, but the RaidendApp doesn't work properly in your web-browser". iOS Chrome app works fine though

## Additional Infos (Optional) 
```
- Browser & version: Metamask browser v2.2.0 (613) and safari browser
- Browser console errors: "We're sorry, but the RaidendApp doesn't work properly in your web-browser"
- Device / OS: iOS 14.4.2
```

**Comment by drselump14** [comment]
Disabling this line make it work though

https://github.com/raiden-network/light-client/blob/4d1d75fd3ac6c56941f1e5676c54bca7cd646c83/raiden-dapp/public/index.html#L25

**Comment by taleldayekh** [comment]
![mm](https://user-images.githubusercontent.com/43838780/120512903-0932c900-c3cc-11eb-9ae5-d1dfe9ddfd6d.jpg)

**Comment by weilbith** [comment]
@drselump14 Hey 👋🏾 
I'm really sorry that we did not respond for such a long time. Initially we were in stress about the mainnet release and then it got a lost for a while. 😓 

I just checked what specific feature is missing in your described setup. It is the [ServiceWorker](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers) which is not available. Unfortunately this is a requirement for the dApp. I have no clue why MetaMask has disabled it (I assume the underlying Chrome engine must have it).

As an alternative you might wanna checkout our new connection methods for the dApp. One of them allows you to use your MetaMask via WalletConnect. Your key and assets remain safe in your wallet application (can be any [that supports](https://registry.walletconnect.org/wallets) WalletConnect). Blockchain interactions in the dApp will then require your confirmation in the connected wallet. You might wanna try this solution. It works between desktop and mobile or mobile exclusively.
